### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -8,7 +8,72 @@
 # @example
 #   include puppetwebhook
 #
+# @example Specify parameters
```
Comment (bastelfreak) [inline manifests/init.pp]: since there are so many docs, did you check if a proper REFERENCE.md can be generated?

Comment (Dan33l) [reply manifests/init.pp]: I will be happy with a `REFERENCE.md` file generated by `puppet strings` command.
@dhollinger  If i can help about this , ping me on IRC.

Excepted this the PR LGTM

**Comment by dhollinger** [comment]
@ekohl re-ran the job and it passed

File: manifests/init.pp (hunk 1 above)
Comment (dhollinger) [reply manifests/init.pp]: REFERENCE. Me should be there now

**Pull request merged**